pyvera/__init__.py

```diff
@@ -284,10 +284,25 @@ def __init__(self, json_obj, vera_controller):
             else:
                 self.name = 'Vera Device ' + str(self.device_id)
 
-    def get_payload_parameter_name(self, name):
+    @staticmethod
```

Comment:
My python probably isn't better than yours: `pylint` was complaining about this method since it doesn't use `self`. I'll back out the change.